docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -622,7 +622,7 @@ part of the server.
 
 When using Vault as a backend you can share configuration with
 all applications by placing configuration in
-`{backend}/application`.  For example, if you run this Vault command
+`secret/application`.  For example, if you run this Vault command
```

Comment:
Hum must be some weird variable replacement issue.  

I cant remember off the top of my head but I think it could be any backend not just `secret`, I think that was the purpose of `{backend}`.  I think what we want to change it to is `\{backend}`.